### ISSUE TITLE
Use of Unicode icons within CSS creates undefined behaviour and general weirdness

### ISSUE THREAD

**Issue by Crazypersonalph** [BetterSEQTA/BetterSEQTA-Plus#57]
Good afternoon everyone,
In my testing of BetterSEQTA+ and in my bugfixing, I have found that the use of unicode icons within the project creates undefined behaviour, that appears sometimes, but disappears other times.

Sometimes, said unicode icons such as the delete icon disappear and are replaced with the unicode icon not found placeholder.

I have found that its occurrence fluctuates, and because of this, I couldn't take a screenshot of it.

I believe the best course of action would be to replace the unicode icons with SVG images, as they have proven to be much more stable

Kind Regards

**Comment by Crazypersonalph** [comment]
Found examples: 
![image](https://github.com/SethBurkart123/EvenBetterSEQTA/assets/93847055/cfd4edb0-37bf-4667-a6bf-58c6c6e93939)

![image](https://github.com/SethBurkart123/EvenBetterSEQTA/assets/93847055/96e543e1-0388-4e69-bff9-0ae8ee159f1d)

![image](https://github.com/SethBurkart123/EvenBetterSEQTA/assets/93847055/b723c0c9-04dc-411b-a2e3-2f93c1f8ca18)

**Comment by SethBurkart123** [comment]
Absolutely, your suggestion about using SVG images instead of unicode icons in BetterSEQTA+ makes a lot of sense. Originally, I opted for font-based icons primarily for their simplicity in CSS integration. However, this approach does have its drawbacks, particularly when it comes to color customization, which can get quite complex. Although it was a practical solution at the time, I'm open to exploring SVG images, given their stability and consistency.

**Comment by SethBurkart123** [comment]
I am currently using a closed source icon library, I can add all of the svg's to the project if you'd like. You can play around with a bit of an implementation.

**Comment by Crazypersonalph** [comment]
> I am currently using a closed source icon library, I can add all of the svg's to the project if you'd like. You can play around with a bit of an implementation.

Is it FontAwesome?

**Comment by SethBurkart123** [comment]
But the question is @Crazypersonalph,

Is it worth replacing with javascript? I know that we could load the svg icons using `background-image`. Before now it wasn't reasonable, but I modified dark mode to instead place a class `.dark` on the main html element, so it might be feasible?

**Comment by ar-cyber** [comment]
Has this been patched? I cannot recreate it.

**Comment by SethBurkart123** [comment]
It's still there, but only occasionally shows up- that's what makes it quite difficult.

**Comment by ar-cyber** [comment]
I think this is stale and should be closed as not planned because of the inactivity.

**Comment by Crazypersonalph** [comment]
Will probably keep open since it is still relevant and will probably on the to-do in the future.

**Comment by StroepWafel** [comment]
won't close it then
